src/keysim.js

```diff
@@ -219,6 +221,14 @@ export class Keyboard {
         if (events & KeyEvents.INPUT) {
           const textinputEvent = this.createEventFromKeystroke('textInput', keystroke, target);
           target.dispatchEvent(textinputEvent);
+          const inputEvent = this.createEventFromKeystroke('input', keystroke, target);
```

Comment:
The `input` event is not exactly the same as `textInput`. It doesn't have a `data` attribute.